modal/app.py

```diff
@@ -586,17 +592,14 @@ def function(
         # Or, pass (request, limit) to additionally specify a hard limit in MiB.
         memory: Optional[Union[int, tuple[int, int]]] = None,
         ephemeral_disk: Optional[int] = None,  # Specify, in MiB, the ephemeral disk size for the Function.
+        min_containers: Optional[int] = None,  # Minimum number of containers to keep warm, even when Function is idle.
```

Comment:
In a way it could be nice if these shared a common prefix (instead of suffix) for in-editor discoverability when typing, but `containers_min` sounds so much worse that I don't know what's best...

Reply:
Yeah `container_floor` / `container_ceiling` is one way you could spell that but it would be sort of bizarre.

Most users won't be typing though!